src/cmd/validate-templates.ts

```diff
@@ -175,16 +175,18 @@ export const validateTemplates = async (): Promise<void> => {
   )
   ;`${kubevalOutput.stdout}\n${kubevalOutput.stderr}`.split('\n').forEach((x) => {
     if (x === '') return
-    const [k, v] = x.split(' - ')
+    const [left, right] = x.split(' - ')
```

Comment:
Looks like a good candidate for unites :)

Reply:
Ah, can you add a suggestion here?